docker-credential-ecr-login.1

```diff
@@ -0,0 +1,91 @@
+.\" Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+.\"
+.\" Licensed under the Apache License, Version 2.0 (the
+.\" "License"). You may not use this file except in compliance
+.\" with the License. A copy of the License is located at
+.\"
+.\"      http://aws.amazon.com/apache2.0/
+.\"
+.\" or in the "license" file accompanying this file. This file is
+.\" distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+.\" CONDITIONS OF ANY KIND, either express or implied. See the
+.\" License for the specific language governing permissions and
+.\" limitations under the License.
+.TH DOCKER-CREDENTIAL-ECR-LOGIN 1 2018-10-29 AMAZON-WEB-SERVICES AMAZON-ELASTIC-CONTAINER-REGISTRY
+.SH NAME
+docker\-credential\-ecr\-login \- Amazon ECR Credential Helper for Docker
+.SH SYNOPSYS
+.B docker\-credential\-ecr\-login
+COMMAND
+
+.B docker\-credential\-ecr\-login
+-v
+.SH DESCRIPTION
+The Amazon ECR Credential Helper for Docker is a credential helper for the
+.IR docker(1)
+command that makes it easier to use Amazon Elastic Container Registry.
+.SH USAGE
+Place the
+.B docker\-credential\-ecr\-login
+binary on your
+.IR PATH
+and set the contents of your
+.IR ~/.docker/config.json
+to be
+\fI{"credsStore":"ecr-login"}\fP.
```

Comment:
I think this snippet of configuration would be more accessible for users in its own code section like the `credHelper` snippet below.

Reply:
Done